css/style.css

```diff
@@ -1573,7 +1573,7 @@ p {
   line-height: 1.4;
 }
 
-@media (min-width: 768px) {
+@media (min-width: 640px) {
```

Comment:
jak używasz media queries to lepiej bedzie uzyc mixinow od bootstrapa niz arbitralnie nadwac minimalna szerokosc. Lepiej tez nie powtarzac sie zrobic blok dla kazdej rozdzielczosci i tam zawrzec wszystkie style zeby tez zachowac kaskadowosc